graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/FederatedSchemaAutoConfiguration.kt

```diff
@@ -86,4 +89,20 @@ class FederatedSchemaAutoConfiguration {
 
         return schema
     }
+
+    /**
+     * Instrumentation is automatically added to the schema if it is registered as a spring component.
+     * This registers the federation tracing instrumentation for federated services.
+     */
+    @Bean
+    @ConditionalOnMissingBean
```

Comment:
should this be conditional on property instead?

Reply:
It is conditional, are you asking if we should not make it conditional? I thought that if someone wanted to provide a different tracing instrumentation with the debug options enabled they could do so by overriding the bean

Reply:
I was thinking more what if I don't want this tracing to be on? i.e. bean is created if no `FederatedTracingInstrumentation` bean exist but what if I don't use Apollo and have my own custom instrumentation instead?